src/Core/SettingsChangesHistory.cpp

```diff
@@ -58,6 +58,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"distributed_cache_read_request_max_tries", 20, 10, "Changed setting value"},
             {"distributed_cache_connect_max_tries", 20, 5, "Changed setting value"},
             {"opentelemetry_trace_cpu_scheduling", false, false, "New setting to trace `cpu_slot_preemption` feature."},
+            {"enable_add_distinct_to_in_subqueries", false, true, "New setting to reduce the size of temporary tables transferred for distributed IN subqueries."},
```

Comment:
```suggestion
            {"enable_add_distinct_to_in_subqueries", false, false, "New setting to reduce the size of temporary tables transferred for distributed IN subqueries."},
```
This setting is disabled by default